vm/tests/solutions/spec/general-purpose-cloud-vm/stackdriver_spec.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2018 Google LLC
```

Comment:
Can you make link to https://github.com/GoogleCloudPlatform/click-to-deploy/blob/master/vm/tests/solutions/spec/__common__/stackdriver_installed.rb instead?

Reply:
done